docs/compatibility.md

```diff
@@ -82,6 +82,6 @@ The following apps are supported by govuk-docker to some extent.
    - ✅ transition
    - ✅ travel-advice-publisher
    - ⚠ whitehall
-      * Who knows, really - several tests are failing, lots pass ;-)
+      * Most tests pass, but several tests fail consistently due to issues with govuk-docker
```

Comment:
```suggestion
      * `make whitehall` requires that you have `publishing-api` running in another shell.
      * It also errors at the `taxonomy:populate_end_to_end_test_data` stage, but you can still `govuk-docker up`.
      * Most tests pass, but several tests fail consistently... (however you want to phrase this)
```

^ Based on a lot of trial-and-erroring this morning while trying to set up smart-answers (which somewhat confusingly has whitehall as a dependency, and kept breaking at the `make` stage as a result. But I was able to start it up regardless).